index.js

```diff
@@ -0,0 +1,35 @@
+const countButton = document.getElementById("countButton")
+countButton.addEventListener("click", (e) => {
+    const userText = document.getElementById("userText").value
+    if(userText.trim()) {
+        const lettersCount = userText.replace(/[^A-Za-zА-Яа-я]/gu, '').length;
+        const wordsCount = userText.split(/\s+/).filter((word) => word.length > 0).length;
+        const sentencesCount = userText.split(/[.!?]+/).filter((sentence) => sentence.length > 0).length;
```

Comment:
The sentence logic is not quiet correct։ in some cases the results gives you one redundant sentence more than it should be. For example if you type "2.5" the sentences count is 2 which is incorrect.